sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/RecognizePiiEntitiesAction.java

```diff
@@ -0,0 +1,162 @@
+    private boolean disableServiceLogs;
+    private PiiEntityDomain domainFilter;
+    private Iterable<PiiEntityCategory> categoriesFilter;
+
+    /**
+     * Get the name of action.
+     *
+     * @return the name of action.
+     */
+    public String getActionName() {
+        return actionName;
+    }
+
+    /**
+     * Set the custom name for the action.
+     *
+     * @param actionName the custom name for the action.
+     *
+     * @return The {@link RecognizePiiEntitiesAction} object itself.
+     */
+    public RecognizePiiEntitiesAction setActionName(String actionName) {
+        this.actionName = actionName;
+        return this;
+    }
+
+    /**
+     * Gets the version of the text analytics model used by this operation.
+     *
+     * @return The model version.
+     */
+    public String getModelVersion() {
+        return modelVersion;
+    }
+
+    /**
+     * Set the model version. This value indicates which model will be used for scoring, e.g. "latest", "2019-10-01".
+     * If a model-version is not specified, the API will default to the latest, non-preview version.
+     *
+     * @param modelVersion The model version.
+     *
+     * @return The {@link RecognizePiiEntitiesAction} object itself.
+     */
+    public RecognizePiiEntitiesAction setModelVersion(String modelVersion) {
+        this.modelVersion = modelVersion;
+        return this;
+    }
+
+    /**
+     * Get the value of {@code includeStatistics}.
+     *
+     * @return The value of {@code includeStatistics}.
+     */
+    public boolean isIncludeStatistics() {
+        return includeStatistics;
+    }
+
+    /**
+     * Set the value of {@code includeStatistics}. The default value is false by default.
+     * If set to true, indicates that the service should return document and document batch statistics
+     * with the results of the operation.
+     *
+     * @param includeStatistics If a boolean value was specified in the request this field will contain
+     * information about the document payload.
+     *
+     * @return The {@link RecognizePiiEntitiesAction} object itself.
+     */
+    public RecognizePiiEntitiesAction setIncludeStatistics(boolean includeStatistics) {
+        this.includeStatistics = includeStatistics;
+        return this;
+    }
+
+    /**
+     * Get the value of {@code disableServiceLogs}.
+     *
+     * @return The value of {@code disableServiceLogs}. The default value of this property is 'false'. This means,
+     * Text Analytics service logs your input text for 48 hours, solely to allow for troubleshooting issues. Setting
+     * this property to true, disables input logging and may limit our ability to investigate issues that occur.
+     */
+    public boolean isServiceLogsDisabled() {
+        return disableServiceLogs;
+    }
+
+    /**
+     * Set the value of {@code disableServiceLogs}.
+     *
+     * @param disableServiceLogs The default value of this property is 'false', except for methods like
```

Comment:
Consider changing the docstrings for both Get and Set here as the default value is true. See .NET as an example: https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/textanalytics/Azure.AI.TextAnalytics/src/RecognizePiiEntitiesAction.cs#L36

Reply:
Good catch. I copied from another option bad but forget PII is different